yarpl/include/yarpl/flowable/Subscriber.h

```diff
@@ -135,6 +135,10 @@ class BaseSubscriber : public Subscriber<T>, public yarpl::enable_get_ref {
 
   virtual void onTerminateImpl() {}
 
+  std::shared_ptr<Subscription> getSubscription() {
```

Comment:
`BaseSubscriber` exposes all methods of `Subscription` on it directly, does there need to be a getter to expose it further?